### ISSUE TITLE
i found the npm install fix

### ISSUE THREAD

**Issue by 88trip** [HashLips/hashlips_art_engine#489]
felt good runnin thru the comments trying to find a solution
so i downgraded from v2022 to v2019
reinstalled the node, and repaired it 
instead of uninstalling it or deleting it entirely
lastly i opened my file set again an to some avail
the process worked seamlessly 

im running windows 11 dev edition 
any questions shoot me a message or comment an i'll do my best to help u all 
cheers

**Comment by JasonJvV** [comment]
perhaps you could dumb it down a little more for me
how did you downgrade the node to v2019?

**Comment by JasonJvV** [comment]
Ok Ive found another solve. I ran version 14 from 2019 without having to delete the node. Follow the following commands to run an older version. (https://www.surrealcms.com/blog/how-to-upgrade-or-downgrade-nodejs-using-npm.html)

**Comment by 88trip** [comment]
Pardon my late reply 
Thats what i most post stated lol 
I downgraded Visual Studio 2019 
An it ran no problems so i believe its an integration issue with the node an the new configurations of the updated node
